CHANGELOG.md

```diff
@@ -24,6 +24,10 @@
   or other error.  
   [Martin Redington](https://github.com/mildm8nnered)
   [#6052](https://github.com/realm/SwiftLint/issues/6052)
+* Fix false positives of `redundant_discardable_let` rule in `@ViewBuilder` functions,
+  `#Preview` macro bodies and preview provides when `ignore_swiftui_view_bodies` is
+  enabled.  
+  [kaseken](https://github.com/kaseken)
```

Comment:
Please add a link to the issue as well.

Reply:
Added issue link here: https://github.com/realm/SwiftLint/pull/6075/commits/3da5c81f8282731a88fe33c5349c8fa8939fd006

Reply:
Added issue link.
https://github.com/realm/SwiftLint/pull/6075/commits/103a1c079bf45046ef3ce5872ed63cb58532099a